torch/csrc/jit/interpreter.cpp

```diff
@@ -611,6 +611,7 @@ struct Instruction {
   UseList inputs;
   ListHandle<int> outputs;
   Symbol debug_name; // used in dump to understand the generated code
+  std::shared_ptr<SourceLocation> debug_location; // for error reporting
 };
```

Comment:
It's a minor thing, but at some point we could isolate the debug information in a parallel list, so that we scan a contiguous chunk during normal execution

Reply:
agreed, I am keeping it simple for now.